packages/ozone-material/ozone-item-preview/src/ozone-item-preview/ozone-item-preview.ts

```diff
@@ -97,12 +96,14 @@ export class OzoneItemPreview extends Polymer.Element {
 		this.set('previewImage', undefined)
 		if (data && data.type) {
 			const typeCache = await getDefaultClient().typeClient().getTypeCache()
-			const isTypeInstanceOf = typeCache.isTypeInstanceOf(data.type, 'media')
-			if (isTypeInstanceOf) {
+			const isMedia = typeCache.isTypeInstanceOf(data.type, 'media')
+			if (isMedia) {
 				const media: Video = data as Video
```

Comment:
Do not cast an item as Video if we're not certain it is one. We should verify it explicitly